[WIP] Prague 2020 full website (#1180)
writethedocs/www: docs/conf/prague/2020/welcome-wagon.rst

:template: {{year}}/generic.html
:banner: _static/conf/images/headers/group.png

Welcome Wagon
=============

Hi! If you're new to Write the Docs, you're in the right place. The Welcome Wagon is here to help out new attendees, 
with both the information on this page, and events we run during the conference.

On this page you'll find important stuff that will help you navigate the conference like a pro,
make you feel more at home, and help you to manage the constant flow of information.

.. image:: /_static/conf/images/pics/welcome-wagon.jpg

> Dururing the conference, we'll connect you with people - both veterans and other first-timers.

Author: ```suggestion
During the conference, we'll connect you with people - both veterans and other first-timers.
```

And we'll give you strategies and tips so you can make the most of the conference.

The `FAQs <#faq>`_ should have answers to most of your questions, but let us know if there is anything you'd like to know that isn't there. You can ask us questions in the :doc:`#wtd-conferences channel on Slack </slack>`.

Who we are
-----------

Hi - we're Beth and Daniel, your Welcome Wagon coordinators! We're really glad you're coming to the {{year}} Write the Docs conference.

If you need a hand at all, we're here to help.

Welcome Wagon events
--------------------

**Write the Docs Introduction**

> *{{date.day_two.dotw}}, {{date.day_two.date}} at 17:00 in the downstairs foyer (near the cloakroom) at Auto-Klub*

Author: ```suggestion
*{{date.day_two.dotw}}, {{date.day_two.date}} at 17:00 in the downstairs foyer (near the cloakroom)*
```

Join us for an informal introduction to Write the Docs, to the Welcome Wagon, and to other first-time conference attendees. We'll pass on some information about the conference specifically for first-timers and give everyone a chance to meet someone new before we join the opening reception.

**Venue Tours**

> * *{{date.day_two.dotw}}, {{date.day_two.date}} at 17:30 starting near the registration desk at Auto-Klub*

Author: ```suggestion
* *{{date.day_two.dotw}}, {{date.day_two.date}} at 17:30 starting near the registration desk*
```

> * *{{date.day_three.dotw}}, {{date.day_three.date}} at 9:15 starting near the registration desk at Auto-Klub*

Author: ```suggestion
* *{{date.day_three.dotw}}, {{date.day_three.date}} at 9:15 starting near the registration desk*
```

Come on a short tour of the venue with a veteran Write the Docs attendee so you'll know where everything is and everything you can take part in.

Pro tips
--------

-  You don't need to go to every talk. Look through the schedule of
   events before you arrive or while you are eating or taking a break.
   Figure out which talks you want to see the most. Spread out your time
   between talks, unconference sessions, networking, and breaks.
-  Speaking of breaks--conferences are exhilarating, but can also be
   exhausting. Give your brain a break! Grab a spot in the quiet room
   or take a quick walk. Play a board game on your lunch break. Come
   back invigorated.
-  Starting Monday morning, check the unconference schedule in the unconference
   room to see if there are any sessions you are interested in
   attending. New sessions are added all the time, so check back
   periodically.
-  Eat! You can use the energy.
-  Are you looking for a job or is there an opening at your company?
   Check out the job board in the unconference room.

FAQ
---

.. contents::
   :local:

Where is everything?
~~~~~~~~~~~~~~~~~~~~

> The conference main stage is in the Auto Klub on the second floor. The unconference

Reviewer: No it's not ;)
Quite a few other Auto Klub references in the rest of this document too.

> The conference main stage is in the Auto Klub on the second floor. The unconference

Author: ```suggestion
The conference main stage is in the Congress Hall on the second floor. The unconference
```

> takes place directly across the foyer in the adjoining room.

Author: ```suggestion
takes place in the Bistro and adjoining rooms, on the left-hand side of the foyer if you stand in front of the Congress Hall.
```

If you are joining in the boat tour on Saturday, you'll meet the other folks
at the Prague Boats pier no. 5.

How should I get around?
~~~~~~~~~~~~~~~~~~~~~~~~

-  Prague is a very walk-able town. Most of the events are

>    within walking distance of the conference venue.

Author: ```suggestion
   within walking distance or a short public transportation ride away from the conference venue.
```

-  There are good public transportation options and taxi services. Check
   out the `Visiting Prague section of the Write the Docs
   website </conf/{{shortcode}}/{{year}}/visiting/>`__ for
   more info.

How should I dress?
~~~~~~~~~~~~~~~~~~~

-  Praguers have a very eclectic dress style, which means they dress however
   they feel! Our documentarians tend to go for a more casual dress at the
   conference. You'll be meeting business colleagues at this conference,
   though, so neat and comfortable are good dress guidelines.
-  If you are going on the `boat tour on
   Saturday </conf/{{shortcode}}/{{year}}/outing/>`__, be sure
   to bring appropriate out-of-doors clothes and shoes. Prague, like most of
   Europe, experiences some ups and downs in the autumn. The first year we
   were here we had a heat wave, and last year we got rained on, so you
   never know! Layering is usually the way to go.

What will I eat?
~~~~~~~~~~~~~~~~

-  Drinks and food are provided for you during conference days, so you can
   focus on the talks and meeting people and don't have to worry about
   where to get your morning coffee.

> -  Coffee, tea, and water are always available in the Auto Klub.

Author: ```suggestion
-  Coffee, tea, and water are always available in the venue.
```

   Bring a water bottle to make it easier for you to stay hydrated.

> -  Food is provided on Sunday, Monday, and Tuesday in the Auto Klub. There

Author: ```suggestion
-  Food is provided on Sunday, Monday, and Tuesday in the conference venue. There
```

   is a light breakfast, a solid lunch, and snacks on each conference
   day.
-  On Saturday and in the evenings on Monday and Tuesday, explore the food
   options in Prague. Invite someone you just met to join you! If you are
   invited to dinner, say yes! Making connections over dinner is a great way
   to get to know more people.
-  If you need grocery items, there is a BILA supermarket at the Prague train station

>    across the park from Auto Klub.

Author: ```suggestion
-  If you need grocery items, there is an Albert supermarket on Ondříčkova 39, 150m west of the venue.
```

Where should I sit?
~~~~~~~~~~~~~~~~~~~

> -  The Auto Klub will have round tables next to the main stage

Author: ```suggestion
-  The Congress Hall has round tables next to the main stage
```

   and rows of seats behind them. There will also be seats in the mezzanine level.
-  There are no reserved seats; feel free to sit anywhere.
-  If you can, show up early to the conference each morning to grab a
   seat at one of the round tables. Introducing yourself to your
   neighbors is one of the easiest way to meet people.

What should I do during the talks?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> `Conference talks at Auto Klub </conf/{{shortcode}}/{{year}}/speakers/>`__:

Author: ```suggestion
`Conference talks </conf/{{shortcode}}/{{year}}/speakers/>`__:
```

-  The time between talks is for meeting your colleagues or taking a
   break. During the talks, listen and take in as much as you can.
-  There is a lot of great information at this conference, but don't

>    worry if you miss something! All talks are videotaped, so you can

Author: ```suggestion
   worry if you miss something! All talks are recorded, so you can
```
